src/serialize.h

```diff
@@ -808,19 +808,6 @@ class CDataStream
     iterator insert(iterator it, const char& x=char()) { return vch.insert(it, x); }
     void insert(iterator it, size_type n, const char& x) { vch.insert(it, n, x); }
 
-    void insert(iterator it, const_iterator first, const_iterator last)
```

Comment:
Nice that we can remove this. There are surely other unused methods... I have created #50 as a work item to find and remove these.